### PR TITLE
fix(bot): no detailed error message when deploy failed

### PR DESCRIPTION
**Pull request by YitongFeng-git**
fix bug https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/10211407

### CODE REVIEW

**Review by Alive-Fish — APPROVED** [review]

**Review by henzhang-ms — APPROVED** [review]

File: packages/fx-core/src/plugins/resource/bot/azureOps.ts [hunk 1]
```diff
@@ -199,10 +200,12 @@ export class AzureOperations {
     try {
       res = await axios.post(zipDeployEndpoint, zipBuffer, config);
     } catch (e) {
+      Logger.error(e.message);
       throw new ZipDeployError(e);
     }
 
     if (!res || !utils.isHttpCodeOkOrCreated(res.status)) {
+      Logger.error(JSON.stringify(res));
```
Comment (henzhang-ms) [inline packages/fx-core/src/plugins/resource/bot/azureOps.ts:208]: Nit: do we need all the res? shall we just log the error code and message?

Comment (YitongFeng-git) [reply packages/fx-core/src/plugins/resource/bot/azureOps.ts:208]: There is no message in res, only status code.
image

Comment (eriolchan) [reply packages/fx-core/src/plugins/resource/bot/azureOps.ts:208]: Agree that we don't need to print headers and request which are not very helpful for trouble shooting. I think the error message should be in data field. Can we trigger some failure on this zip deploy step (or just ask bug creator how to repro this) and check what's the value of data when status code is not 200?

In general, status code and data is enough to be print

Comment (eriolchan) [reply packages/fx-core/src/plugins/resource/bot/azureOps.ts:208]: Agree that we don't need to print headers and request which are not very helpful for trouble shooting. I think the error message should be in data field. Can we trigger some failure on this zip deploy step (or just ask bug creator how to repro this) and check what's the value of data when status code is not 200?

In general, status code and data is enough to be print

File: packages/fx-core/src/plugins/resource/bot/azureOps.ts [hunk 2]
```diff
@@ -205,7 +205,9 @@ export class AzureOperations {
     }
 
     if (!res || !utils.isHttpCodeOkOrCreated(res.status)) {
-      Logger.error(JSON.stringify(res));
+      Logger.error(
+        `Deploy endpoint: ${zipDeployEndpoint}, status code: ${res.status}, response data: ${res.data}`
```
Comment (eriolchan) [inline packages/fx-core/src/plugins/resource/bot/azureOps.ts:209]: If res.data is JSON format, can it be print correctly?

**Review by eriolchan — APPROVED** [review]

**Pull request merged**